$OpenBSD: patch-configure_ac,v 1.1.1.1 2011/07/04 15:28:05 jasper Exp $

commit 3d43874a54e1308510af28e63c9c3f72e0ad316a
Author: Jasper Lievisse Adriaanse <jasper@humppa.nl>
Date:   Thu Jun 30 15:49:53 2011 +0200

    Recognize OpenBSD so the right uuid library is picked up.
    
    https://bugzilla.gnome.org/show_bug.cgi?id=653745

--- configure.ac.orig	Sat Apr 30 14:35:11 2011
+++ configure.ac	Mon Jul  4 11:30:05 2011
@@ -5,6 +5,7 @@ AC_CONFIG_SRCDIR(libgupnp/gupnp.h)
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 AC_CANONICAL_HOST
+AC_CANONICAL_TARGET
 
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
@@ -17,7 +18,7 @@ AC_FUNC_MMAP
 AC_SYS_LARGEFILE
 
 AC_MSG_CHECKING([operating system])
-case "$host_os" in
+case "$target_os" in
      mingw*|cygwin*)
         AC_MSG_RESULT(Win32)
         UUID=uuid
@@ -26,8 +27,12 @@ case "$host_os" in
         AC_MSG_RESULT(Darwin)
         UUID=
         ;;
+     openbsd*)
+        AC_MSG_RESULT(OpenBSD)
+        UUID=e2fs-uuid
+        ;;
      *)
-        AC_MSG_RESULT($host_os)
+        AC_MSG_RESULT($target_os)
         UUID=uuid
         ;;
 esac
